GVFS/GVFS.FunctionalTests/Tests/GitCommands/ResetMixedTests.cs

```diff
@@ -78,19 +78,6 @@ public void ResetMixedThenCheckoutWithConflicts()
             this.FilesShouldMatchCheckoutOfTargetBranch();
         }
 
-        [TestCase]
-        public void ResetMixedOnlyAddedThenCheckoutWithConflicts()

```

Comment:
I'm confused.  This test case was modified in the second commit to change `Validate...` to `Run...`, but then the whole test was deleted in the third commit.  Not sure if this was intended or what.

Reply:
Sorry for the commit confusion. I could not get this test to work the same because of how the (intended?) behavior changed for sparse paths.

Reply:
While rebasing onto v2.27.0-rc0, I think I found a reason why this test would fail. I'll close this PR and work on the rebased version instead.